prombench/manifests/prombench/benchmark/6_loadgen.yaml

```diff
@@ -48,6 +48,33 @@ data:
         - expr: histogram_quantile(0.99, sum by(path, le) (rate(codelab_api_request_duration_seconds_bucket{method="POST"}[5m])))
         - expr: histogram_quantile(0.99, sum by(path, method, le) (rate(codelab_api_request_duration_seconds_bucket{method="POST"}[5m])))
         - expr: histogram_quantile(0.99, sum by(instance, le) (rate(codelab_api_request_duration_seconds_bucket{method="POST"}[5m])))
+      - name: arithmetic_operation
+        interval: 10s
+        type: instant
+        queries:
+        - expr: rate(codelab_api_request_duration_seconds_bucket{method="POST"}[5m]) + 100  
+        - expr: rate(go_memstats_frees_total[5m]) * 60
+        - expr: rate(codelab_api_requests_total{method=~"GET|POST"}[5m]) - rate(codelab_api_request_duration_seconds_sum{method=~"GET|POST"}[5m])
+        - expr: rate(go_gc_duration_seconds_sum{job=~"fake-webservers-[0-9]+"}[5m]) / rate(go_gc_duration_seconds_count{job=~"fake-webservers-[0-9]+"}[5m])        
+        - expr: sum by (instance, job) (rate(codelab_api_request_errors_total[5m])) / sum by (instance, job) (rate(go_memstats_mallocs_total[10m]))
+      - name: logic_operator
+        interval: 10s
+        type: instant
+        queries:
+        - expr: codelab_api_request_duration_seconds_bucket{method="GET"} or codelab_api_request_duration_seconds_bucket{method="POST"}
+        - expr: codelab_api_request_duration_seconds_sum{status="200"} or codelab_api_request_duration_seconds_sum{status="500"}
+        - expr: codelab_api_request_duration_seconds_bucket{status="200"} and codelab_api_request_duration_seconds_bucket{method="GET"}
+        - expr: codelab_api_request_duration_seconds_count{method="POST"} and codelab_api_request_duration_seconds_count{status="500"}
+        - expr: codelab_api_request_duration_seconds_bucket{status="200"} or codelab_api_request_duration_seconds_bucket{method="GET"}
+      - name: topk
+        interval: 10s
+        type: instant
+        queries:
+        - expr: topk(2000, sum(rate(go_gc_duration_seconds_count[5m])) by (instance, job))
+        - expr: topk(10000, sum(codelab_api_request_duration_seconds_bucket) by (method,job))     
```

Comment:
`topk(10000` is not realistic; nobody is going to scroll down 10,000 lines of screen output to find something.
`k` should be more like 10, or perhaps 100.
Also I don't think there are 10,000 combinations of method and job.
Also it is not valid to sum histogram buckets.